alvr/server/build.rs

```diff
@@ -24,74 +24,57 @@ fn do_ffmpeg_config(build: &mut cc::Build) {
 
     assert!(ffmpeg_path.join("include").exists());
     build.include(ffmpeg_path.join("include"));
-
-    #[cfg(all(feature = "gpl", target_os = "linux"))]
-    {
-        let ffmpeg_pkg_path = ffmpeg_path.join("lib").join("pkgconfig");
-        assert!(ffmpeg_pkg_path.exists());
-
-        let ffmpeg_pkg_path = ffmpeg_pkg_path.to_string_lossy().to_string();
-        env::set_var(
-            "PKG_CONFIG_PATH",
-            env::var("PKG_CONFIG_PATH").map_or(ffmpeg_pkg_path.clone(), |old| {
-                format!("{ffmpeg_pkg_path}:{old}")
-            }),
-        );
-
-        let pkg = pkg_config::Config::new().cargo_metadata(false).to_owned();
-
-        let avutil = pkg.probe("libavutil").unwrap();
-        let avfilter = pkg.probe("libavfilter").unwrap();
-        let avcodec = pkg.probe("libavcodec").unwrap();
-        let swscale = pkg.probe("libswscale").unwrap();
-
-        build
-            .define("AVCODEC_MAJOR", avcodec.version.split(".").next().unwrap())
-            .define("AVUTIL_MAJOR", avutil.version.split(".").next().unwrap())
-            .define(
-                "AVFILTER_MAJOR",
-                avfilter.version.split(".").next().unwrap(),
-            )
-            .define("SWSCALE_MAJOR", swscale.version.split(".").next().unwrap());
-
-        // activate dlopen for libav libraries
-        build
-            .define("LIBRARY_LOADER_AVCODEC_LOADER_H_DLOPEN", None)
-            .define("LIBRARY_LOADER_AVUTIL_LOADER_H_DLOPEN", None)
-            .define("LIBRARY_LOADER_AVFILTER_LOADER_H_DLOPEN", None)
-            .define("LIBRARY_LOADER_SWSCALE_LOADER_H_DLOPEN", None);
-
-        println!("cargo:rustc-link-lib=dl");
-    }
 }
 
 fn do_ffmpeg_config_post() {
     if cfg!(feature = "local_ffmpeg") {
-        // TODO: cfg!(feature = "gpl") - switch to static linking
-        let kind = if false { "static" } else { "dylib" };
+        let statik = cfg!(all(feature = "gpl", target_os = "linux"));
```

Comment:
Misspelled static.

Reply:
static is a keyword. pkg_config already uses statik so this should be fine.

Reply:
Ah, you you still can use that with i think r#

Reply:
I see misspelled words all the time to avoid writing reserved words. It's fine, although it looks a bit nerdy, i usually try to find some synonym (but it's not always possible).

Reply:
What I usually do is suffix it with a `_`, but pkg_conf already used `statik` so I went with that.